excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/util/mapper/SheetName.java

```diff
@@ -27,7 +27,8 @@
     AUTHORISATION_CASE_EVENT("AuthorisationCaseEvent"),
     AUTHORISATION_CASE_STATE("AuthorisationCaseState"),
     SEARCH_ALIAS("SearchAlias"),
-    BANNER("Banner");
+    BANNER("Banner"),
+    NOC_CONFIG("NoCConfig");
```

Comment:
Hi Kiran. Can we make the name of the tab more descriptive? i.e NoticeOfChangeConfig, instead of NoCConfig.

Reply:
chanded NoCConfig tab to NoticeOfChangeConfig tab